zwave_js/rootfs/etc/cont-init.d/config.sh

```diff
@@ -67,6 +75,12 @@ if [[ ${flush_to_disk:+x} ]]; then
     bashio::addon.options > "/data/options.json"
 fi
 
+# We have to reflush the config to disk if we have set the network_key
+if ! bashio::config.has_value 'network_key'; then
+    bashio::addon.option 'network_key' "$(bashio::config 's0_legacy_key')"
+    bashio::addon.options > "/data/options.json"
+fi
```

Comment:
The if check should be moved up, and should set `flush_to_disk=1` instead of writing directly here

Reply:
the problem was that I couldn't access the newly set option until I flushed to disk the first time to handle the case where `s0_legacy_key` is unset initially

Reply:
With that being said, I can move this into the auto generation logic so we only flush once